lib/explorer/data_frame.ex

```diff
@@ -5553,6 +5554,11 @@ defmodule Explorer.DataFrame do
   When the dataframe has grouping variables, operations are performed per group.
   `Explorer.DataFrame.ungroup/2` removes grouping.
 
+  ## Options
+
+    * `:stable` (`boolean`) — when `true`, preserve the order of the groups exactly as they appear in the original dataframe.
+      Groups may be reordered (for example, sorted) for performance. By default `false`.
```

Comment:
What if we pass the stable option when aggregating instead?

Reply:
s/aggregate/summarize

Reply:
Right, but they may be "unstable" in the future... this will give us the opportunity to deal with those later (otherwise doing stable when unstable is required is fine).

My thinking is that instability is a property of the dataframe? Usually if you want to keep it stable, you would keep it throughout all ops?